functorch/functorch/_src/compilers.py

```diff
@@ -142,6 +142,7 @@ def nnc_jit(f, static_argnums=None):
 default_decompositions = get_decompositions(default_decompositions)
 
 
+@make_boxed_compiler
```

Comment:
Just checking -- this changes the semantics of print_compile, right?

Reply:
I don't think so. What semantics do you think it's changing?